### DISCUSSION TITLE
Why bother with UML at all when modeling the ontology?

### DISCUSSION THREAD

**Discussion by jgmikael** [eprocurementontology/eprocurementontology#441] [General]
Hi, I watched the ePO presentation at Endorse by Natalie Muric. Really cool to see OWL & SHACL based ontologies developed that could even be referenced to as Linked Data once there's content negotiation on the server and the URIs in the namespace are resolvable.

Still a bit puzzled why you feel the need for having business users to model things first in UML? In the Finnish governmental semantic work we introduced a user friendly RDF/OWL/SHACL modeling environment already years ago. Why bother with adding extra complexity to the work?

**Comment by SellittoGiampaolo** [comment]
It is likely that the most used format for the ontology up to now is its UML representation, especially when we deal with business users.
While we know that UML could imply the acceptance of some trade-off and simplification if compared to RDF/OWL/SHACL, it remains the most usable way to visually present concepts and relations in the domain of Public Procurement. 
The importance of visualization is out of discussion, but when designing the ePO ontology also usability and interoperability were taken into account and in that respect the use of UML represented a key factor for its adoption both by business people and by developers, who are not at ease in using RDF/OWL/SHACL modeling environments.

**Comment by WvandenBerg** [comment]
> In the Finnish governmental semantic work we introduced a user friendly RDF/OWL/SHACL modeling environment already years ago. Why bother with adding extra complexity to the work?

I'm curious. Is there some way I can have a look at this modeling environment?

**Reply by jgmikael** [reply]
Sure: documentation in English > https://wiki.dvv.fi/display/YTIJD/Finnish+interoperability+platform+and+tools > you can access the tools through links in the documentation. 
Source code: https://github.com/VRK-YTI
